src/main/resources/templates/rooms/layout.html

```diff
@@ -28,8 +28,17 @@ <h1 style="margin-left: 0.42em; margin-top: 0.15em">
     </div>
```

Comment:
@MorMundHS-MA I would assume that this checkout-bar div and its CSS definition in main.css can be savely removed, can't it?